proxy/src/main/java/com/wavefront/agent/PointHandlerImpl.java

```diff
@@ -39,6 +39,7 @@
   private final Histogram receivedPointLag;
   private final String validationLevel;
   private final int port;
+  private static long discardPointsHours = 8760; // Default one year
```

Comment:
if this is a constant, if should be

`private static final long DISCARD_POINT_HOURS = 8760`